openmetadata-service/src/test/java/org/openmetadata/service/resources/lineage/LineageResourceTest.java

```diff
@@ -327,6 +327,16 @@ void putLineageFromDashboardDataModelToTable() throws HttpResponseException {
         "Column level lineage is only allowed between two tables or from table to dashboard.");
   }
 
+  @Order(5)
+  @Test
+  void put_lineageWithDescription() throws HttpResponseException {
```

Comment:
@07Himank please add a test for patch as well

Reply:
Yeah sure will do it.